paddle/cinn/ir/tensor.cc

```diff
@@ -553,14 +553,14 @@ ir::Tensor _Tensor_::Reshape(const std::vector<Expr> &shape,
   auto selft = Tensor(const_cast<ir::_Tensor_ *>(this));
 
   {
-    Expr this_num_elements = Expr(1);
-    for (auto &e : this->shape) this_num_elements = this_num_elements * e;
+    int32_t this_num_elements = 1;
+    for (auto &e : this->shape)
```

Comment:
Try to add braces for `for` loops even if they only contain one statement.

Reference:

https://google.github.io/styleguide/cppguide.html#Formatting_Looping_Branching

Reply:
Done✅